packages/web-components/src/components/masthead/__stories__/masthead.stories.ts

```diff
@@ -286,6 +287,28 @@ withAlternateLogoAndTooltip.story = {
   },
 };
 
+export const WithCloudVariant = ({ parameters }) => {
+  const { selectedMenuItem, userStatus, searchPlaceholder, hasProfile, hasSearch } = parameters?.props?.MastheadComposite ?? {};
+  return html`
+    <style>
+      ${styles}
+    </style>
+    <dds-cloud-masthead-container
+      platform="Cloud"
+      platform-url="${ifNonNull(platformData.url)}"
+      selected-menu-item="${ifNonNull(selectedMenuItem)}"
+      user-status="${ifNonNull(userStatus)}"
+      searchPlaceholder="${ifNonNull(searchPlaceholder)}"
+      .authenticatedProfileItems="${ifNonNull(authenticatedProfileItems)}"
+      .navLinks="${customLinks}"
+      ?has-profile="${hasProfile}"
+      ?has-search="${hasSearch}"
+      .unauthenticatedProfileItems="${ifNonNull(unauthenticatedProfileItems)}"
+      data-endpoint="/common/carbon-for-ibm-dotcom/translations/cloud-masthead"
```

Comment:
was double checking why the endpoint was resulting in a 404 - turns out I forgot we need to add /jsononly to the path. We should be able to get the data using /common/carbon-for-ibm-dotcom/translations/cloud-masthead/jsononly

Reply:
@annawen1 This is good to know. Thanks for pointing this out. Also, since we're using the JSON-only directory, I'm guessing we don't need the `.js` within the dotcom translation repo? https://github.ibm.com/webstandards/ibm-dotcom-translations/blob/master/translations/cloud-masthead/usen.js

Reply:
I just wanted to bring this up as this will play in our translation content population workflow and if we're only using the JSON files that would help come down in the formatting of the `.js` versions. Let me know what you think.

Reply:
Yeah we don't need the `.js` file for the translations, I think we initially kept it for consistency when copying over the current ibm.com translations folder structure? Not sure if removing it will cause any issues. @jeffchew what do you think?

Reply:
@annawen1 Thanks! @jeffchew Let me know if we need to account for the `.js` files within the Cloud Masthead translation directory.